x-pack/packages/kbn-slo-schema/src/rest_specs/slo.ts

```diff
@@ -230,7 +235,20 @@ const deleteSLOInstancesParamsSchema = t.type({
 
 const fetchHistoricalSummaryParamsSchema = t.type({
   body: t.type({
-    list: t.array(t.type({ sloId: sloIdSchema, instanceId: t.string })),
+    list: t.array(
+      t.intersection([
+        t.type({
+          sloId: sloIdSchema,
+          instanceId: t.string,
+          timeWindow: timeWindowSchema,
+          budgetingMethod: budgetingMethodSchema,
+          objective: objectiveSchema,
+          groupBy: allOrAnyStringOrArray,
+          revision: t.number,
+        }),
+        t.partial({ remoteName: t.string }),
+      ])
+    ),
```

Comment:
I haven't seen the update in the historical API yet, but If possible we should keep the previous payload working for local SLOs (which is probably the main use case), and the other optional fields for remote SLOs, e.g. `t.union([localFields, remoteFields])`.
Again, without looking at the API changes yet, I wonder if we have to provide all these settings, or if the optional remoteName is sufficient.